admin/src/components/Navbar.vue

```diff
@@ -60,15 +61,15 @@ export default {
         .signOut()
         .then(
           () => {
-            this.$router.push({ name: "Login" });
+            this.$router.push({ name: 'Login' });
           },
-          error => {
+          (error) => {
```

Comment:
can you ensure your linter doesn't parenthesize single params, we're following the Airbnb styleguide. https://github.com/airbnb/javascript#arrows--one-arg-parens